BaseTools/Plugin/HostBasedUnitTestRunner/HostBasedUnitTestRunner.py

```diff
@@ -207,14 +207,14 @@ def gen_code_coverage_msvc(self, thebuilder):
         # Generate coverage file
         coverageFile = ""
         for testFile in testList:
-            ret = RunCmd("OpenCppCoverage", f"--source {workspace} --export_type binary:{testFile}.cov -- {testFile}")
+            ret = RunCmd("OpenCppCoverage", f"--source {workspace} --export_type binary:{testFile}.cov -- {testFile}", workingdir=f"{workspace}Build/")
             coverageFile += " --input_coverage=" + testFile + ".cov"
             if ret != 0:
                 logging.error("UnitTest Coverage: Failed to collect coverage data.")
                 return 1
 
         # Generate and XML file if requested.by each package
-        ret = RunCmd("OpenCppCoverage", f"--export_type cobertura:{os.path.join(buildOutputBase, 'coverage.xml')} --working_dir={workspace}Build {coverageFile}")
+        ret = RunCmd("OpenCppCoverage", f"--export_type cobertura:{os.path.join(buildOutputBase, 'coverage.xml')} --working_dir={workspace}Build/ {coverageFile}", workingdir=f"{workspace}Build/")

```

Comment:
Is it necessary to set the working dir both via the OpenCppCoverage cli flag and via RunCmd? I looks like either or would work fine.

Reply:
Ah, yes we can probably remove the CLI flag since we are not doing it in the popen.

Reply:
To clarify though, we CLI one is not sufficient since it ignores that flags value for LastCoverageResults.log 😢